pkg/functions/lib/jwt/errors.go

```diff
@@ -0,0 +1,46 @@
+package jwt
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/jexia/semaphore/pkg/specs/types"
+)
+
+var (
+	errInvalidToken       = errors.New("token is invalid")
+	errInvalidValueType   = errors.New("invalid value, expected a string")
+	errMalformedAuthValue = errors.New("malformed auhtorization value")
```

Comment:
These errors should be exported if we want other packages to perform type checks.

Reply:
not sure it is needed
(they were introduced to perform error checks in tests)